### PR TITLE
spec/03: Update directories for scripts

### PR DESCRIPTION
**Pull request by MrKevinWeiss**
Change directory of compile_and_test_for_board.py to proper path
Change directory of tapsetup to proper path

### CODE REVIEW

File: 02-tests/02-tests.md [hunk 1]
```diff
@@ -2,11 +2,12 @@
 
 ### Usage
 
-Firstly, run the tapsetup tool in `dist/tools/tapsetup` if you want to test for
-`native`. Then, run the `RIOT/dist/tools/compile_and_test_for_board.py` script
+Firstly, run the tapsetup tool in `dist/tools/tapsetup/tapsetup` if you want
```
Comment (miri64) [inline 02-tests/02-tests.md]: The tool is not _in_ the path you changed it to. So either keep as is or remove the "in"

Comment (MrKevinWeiss) [reply 02-tests/02-tests.md]: Yup agreed.

Comment (MrKevinWeiss) [reply 02-tests/02-tests.md]: agreed

**Comment by MrKevinWeiss** [comment]
Thanks!

**Pull request merged**